### DISCUSSION TITLE
Select an element with shared properties on same page

### DISCUSSION THREAD

**Discussion by AutomationDevCypress** [cypress-io/cypress#14176] [Component Testing] [answered]
I am trying to select a dropdown value on a page. when inspect i am getting 2 elements with same properties on page and only want to select from the very first dropdown. Following is the elements properties
<select name="WRQSperiod" data-bind="event: { change: loadWidgetData }" css="1">
                    <option value="MTD" data-bind="html: translate('common.mtd')">MTD</option>
                    <option value="YTD" data-bind="html: translate('common.ytd')">YTD</option>
                    <option value="Today" data-bind="html: translate('common.period_today')">Today</option>
                    <option value="Yesterday" data-bind="html: translate('common.period_yesterday')">Yesterday</option>
                    <option value="LastMonth" data-bind="html: translate('common.last_month')">Last Month</option>
                </select>

When inspect i am getting another element on same page with the only difference is css="2"
![image](https://user-images.githubusercontent.com/75641421/102126067-954e3580-3e18-11eb-8882-bb26c798058e.png)

In my code this is what i am trying to achieve
cy.get('select[name="WRQSperiod"]').select('YTD') but i am getting an error
cy.select() can only be called on a single <select>. Your subject contained 2 elements

**Comment by jennifer-shehane** [comment, marked as answer]
You can try selecting the first select of the elements found

```js
cy.get('select[name="WRQSperiod"]').first().select('YTD')
// or
cy.get('select[name="WRQSperiod"]').eq(0).select('YTD')
```

Or by specifying the css value

```js
cy.get('select[name="WRQSperiod"][css="1"]').eq(0).select('YTD')
```

**Reply by AutomationDevCypress** [reply]
Thanks, the first 2 helped me.
